TCMSApp/src/server/model/mschemas/suite.js

```diff
@@ -13,13 +13,24 @@ var testSuiteSchema = new Schema({
     tests: [{
         testName: {type: String, required: true},
         testDescription: {type: String, required: true},
+        suiteId: mongoose.Schema.Types.ObjectId,
         automated: Boolean,
         preConditions: {type: String, required: true},
         steps: [{
             stepNumber: Number,
             stepDescription: {type: String, required: true},
-            expectedResult: {type: String, required: true}
-        }]
+            expectedResult: {type: String, required: true},
+            status: {
```

Comment:
ohh, suites have no statuses. only runs.

suites/tests/steps are scenarios. 
runs can have statuses.

But!
we might have an idea to connect test steps with the result of an execution of all test runs made on the particular test.
It can be done by keeping the reference to array of test runs from the test objects! This fact can be discussed